src/compiler/cpp_generator.cc

```diff
@@ -106,7 +107,7 @@ grpc::string GetHeaderPrologue(File *file, const Parameters & /*params*/) {
     printer->Print(vars,
                    "// If you make any local change, they will be lost.\n");
     printer->Print(vars, "// source: $filename$\n");
-    grpc::string leading_comments = file->GetLeadingComments();
+    grpc::string leading_comments = file->GetComments(true, "//");
```

Comment:
I'm in favor of GetLeadingComments/GetTrailingComments as opposed to a "leading" param